pkg/apis/aws/v1alpha1/types_infrastructure.go

```diff
@@ -35,6 +35,16 @@ type InfrastructureConfig struct {
 	// for details of the underlying terraform implementation.
 	// +optional
 	IgnoreTags *IgnoreTags `json:"ignoreTags,omitempty"`
+
+	// Pods are the CIDRs of the pod network.
+	// +optional
+	Pods []string `json:"pods,omitempty"`
+	// Nodes are the CIDRs of the node network.
+	// +optional
+	Nodes []string `json:"nodes,omitempty"`
+	// Services are the CIDRs of the service network.
+	// +optional
+	Services []string `json:"services,omitempty"`
```

Comment:
I am sorry but I don't see the connection with the new types introduced here. The PR in g/g adds new infrastructure status that you can populate already with whatever is in the `shoot.spec.networking`.

After the IPv6 PR you just need to propagate ranges that you read/created from the infra reconciliation step and that doesn't require anything from the infrastructureConfig right ?